packages/dd-trace/src/id.js

```diff
@@ -34,6 +35,11 @@ class Identifier {
     return this._buffer
   }
 
+  // msgpack-lite compatibility
+  toArray () {
+    return this._buffer.subarray(-8)
```

Comment:
Why is `subarray` needed here?

Reply:
Because the ID can be more than 8 bytes (was done to support B3/W3C).